pkgs/sdk/server/src/LdClient.cs

```diff
@@ -397,6 +397,11 @@ public FeatureFlagsState AllFlagsState(Context context, params FlagsStateOption[
                 {
                     EvaluatorTypes.EvalResult result = _evaluator.Evaluate(flag, context);
                     bool inExperiment = EventFactory.IsExperiment(flag, result.Result.Reason);
+
+                    var directPrerequisites = result.PrerequisiteEvals.Where(
+                        e => e.FlagKey == flag.Key)
+                        .Select(p => p.PrerequisiteFlag.Key).ToList();
```

Comment:
Converting to a list here, rather than leaving as an `IEnumerable`, because the intent is that order is preserved.